Simplenote/Classes/SPSettingsViewController+Extensions.swift

```diff
@@ -88,3 +88,85 @@ private extension BiometricAuthentication.Biometry {
         }
     }
 }
+
+// MARK: - Account Deletion
+extension SPSettingsViewController {
+
+    @objc
+    func deleteAccountWasPressed() {
+        guard let user = SPAppDelegate.shared().simperium.user else {
+            return
+        }
+        SPTracker.trackDeleteAccountButttonTapped()
+        spinnerViewController = SpinnerViewController()
+        spinnerViewController.startAnimating()
+
+        let deletionController = AccountDeletionController()
+        SPAppDelegate.shared().accountDeletionController = deletionController
```

Comment:
I had left this un initialized cause we would only need it if someone had requested an account deletion.  Otherwise the app delegate doesn't need to have an active deletion controller.  

I can set it up, but since initializing didn't require any configuration I felt that setting it up here when the user went to delete was an okay time to create this object, and other wise leave it as nil

WDYT?